### PR TITLE
docs/13583-fullscreen

### PR DESCRIPTION
**Pull request by sebastianbochan**
#13583, docs: fullscreen, added information about limits.

### CODE REVIEW

File: ts/Stock/StockToolsBindings.ts [hunk 1]
```diff
@@ -1994,6 +1994,8 @@ var stockToolsBindings: Highcharts.Dictionary<Highcharts.NavigationBindingsOptio
     /**
      * Displays chart in fullscreen.
      *
+     * Fullscreen is not supported on iPhone due to iOS limits.
```
Comment (pawelfus) [inline ts/Stock/StockToolsBindings.ts:1997]: Could you use bold `**Note**:` at the beginning? To highlight this :)

Comment (sebastianbochan) [reply ts/Stock/StockToolsBindings.ts]: Fixed

**Comment by sebastianbochan** [comment]
Im not sure why test not passed, especially that is only docs edit. It looks like an issue in the branch.

**Comment by sebastianbochan** [comment]
> Shouldn't it be "iOS limitations" rather than "iOS limits"? 🤔

Indeed, will commit.

**Comment by sebastianbochan** [comment]
Done

**Pull request merged**